_includes/components/icon.html

```diff
@@ -1,12 +1,12 @@
 {% comment %}
 include
 - icon
-- size
+- icon_size
 {% endcomment %}
 
 {% assign class = 'usa-icon' %}
-{% if include.size > 1 %}
```

Comment:
I was having trouble with this adding weird unexpected classes like `usa-icon--size-2` even when I wasn't assigning a value, and came to the conclusion it's probably using `size` in the sense of an array or string Ruby object, hence the rename.